docs/release_notes/workspace_prefix.md

```diff
@@ -0,0 +1,17 @@
+## /Workspace prefix
+
+This PR (https://github.com/databricks/cli/pull/1724) introduced the following changes:
```

Comment:
Ideally this doc wouldn't talk about the a PR but would say  something like "As of revision x.y, Databrick Asset Bundles ..."